File: user-service/infra-persistence/src/main/java/com/yellowsunn/userservice/persistence/config/JpaConfiguration.java
package com.yellowsunn.userservice.persistence.config;

import com.yellowsunn.userservice.persistence.PersistenceModule;
import com.yellowsunn.userservice.domain.DomainModule;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EntityScan(basePackageClasses = DomainModule.class)
@EnableJpaRepositories(basePackageClasses = PersistenceModule.class)
public class JpaConfiguration {
}

File: board-service/libs/infra-persistence/src/main/kotlin/com/yellowsunn/boardservice/persistence/config/JpaConfiguration.kt
package com.yellowsunn.boardservice.persistence.config

import com.yellowsunn.boardservice.domain.DomainModule
import com.yellowsunn.boardservice.persistence.PersistenceModule
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@EntityScan(basePackageClasses = [DomainModule::class])
@EnableJpaRepositories(basePackageClasses = [PersistenceModule::class])
class JpaConfiguration
